database/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.6.1",
   "description": "Database layer for TheUsher",
   "scripts": {
-    "test": "mocha",
+    "test": "mocha --exit",
```

Comment:
@k1ch  how come we had to add `--exit` here?

Reply:
Without the `--exit` flag the test process does not exit. Review [this GH check](https://github.com/DMGT-TECH/the-usher-server/actions/runs/7268711434/job/19812416972#step:9:150) which I had to cancel after 1hr 38m